nextflow_schema.json

```diff
@@ -323,11 +349,14 @@
         {
             "$ref": "#/definitions/input_output_options"
         },
+        {
+            "$ref": "#/definitions/trimming"
+        },
         {
             "$ref": "#/definitions/sourmash_options"
         },
         {
-            "$ref": "#/definitions/trimming"
```

Comment:
Don't we need that anymore?

Reply:
this change was made automatically by nf-core schema build, I think